contrib/slime/README.org

```diff
@@ -74,6 +74,15 @@ of Common Lisp, you can specify it in your =~/.spacemacs=
 | ~SPC m c f~ | Compile function         |
 | ~SPC m c r~ | Compile region           |
 
+** Definition
+
+| Key Binding               | Description                     |
+|---------------------------+---------------------------------|
+| ~SPC m d d~               | Go to definition                |
+| ~SPC m d u~ or ~SPC m d D | Return from definition          |
```

Comment:
There is no conventions for this one but I would like to put this pop-stack on ``SPC m g ` `` which matches ``SPC ` `` and ``` `` ```

`SPC m g u` is already taken for `go to usages`.